code-scanning/properties/defender-for-devops.properties.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Microsoft Defender For DevOps Scan",
     "creator": "Microsoft",
-    "description": "Defender for devops helps integrate multiple tools with GitHub Advanced Security and sends the results to Defender for cloud dashbord.",
+    "description": "Defender for devops helps integrate multiple tools with GitHub Advanced Security and sends the results to Defender for cloud dashboard.",
     "iconName": "microsoft",
     "categories": ["Code Scanning", "IaC","Docker Images", "Defender For DevOps", "EsLint", "Bandit", "Trivy", "BinSkim", "TemplateAnalyzer"]
```

Comment:
Can we update this according to categories defined here: https://github.com/actions/starter-workflows#valid-properties

Reply:
Done